ui/src/app/base/components/Header/Header.js

```diff
@@ -213,7 +215,11 @@ export const Header = () => {
               <li className="p-navigation__link" role="menuitem">
                 <a
                   className="p-dropdown__item"
-                  href={generateURL("accounts/logout/")}
+                  href="##"
```

Comment:
Why `##` rather than `#`?

Reply:
To satisfy the linter :)

Reply:
Ah ok. Maybe we should override that rule if we have to hack around it?

Reply:
Yeah good idea, I've filed: https://github.com/canonical-web-and-design/maas-ui/issues/284